views/mixins/sections.pug

```diff
@@ -1,34 +1,34 @@
 include controls.pug
 
-mixin showSectionService(service, plan, instance)
+mixin showSectionService(service, plan, instance, options)
   section#service.section
     .container
       .row
         .col-lg-12
           h4 Service Information
       .row
         .col-lg-6
-           +renderPropertyListPanel('Service', {
+           +renderPropertyListPanel('Service', _.assign({
```

Comment:
you are passing this additional options parameter, but the mixin defintion `renderPropertyListPanel` isnt changed. So how will this value `options.service` be used? Also, isnt `options` an HTML fragment which is to be just embedded? Looks like you are using it as a JSON.

Also it would help if you can checkin a sample blueprint_dashboard template sample as well so that we can see the exact flow play out here.

Reply:
`options` is a JSON object. In `renderPropertyListPanel` we pass title of panel and a json object which is then displayed in tabular format.
https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/views/mixins/controls.pug#L54
You can find a sample `blueprint-dashboard.yml.ejs`  file in pr

Reply:
Ok. Got it, but as discussed let us also evaluate and try to put in an option of embedding an HTML snippet as well from service providers.

Reply:
ok. Got it.